conda_forge_tick/auto_tick.py

```diff
@@ -609,8 +599,8 @@ def add_rebuild_migration_yaml(
     excluded_feedstocks: MutableSet[str],
     exclude_pinned_pkgs: bool,
     migration_yaml: str,
-    config: dict = {},
-    migration_name: str = "",
+    config: dict,
+    migration_name: str,
```

Comment:
Please don't change defaults on functions. The bot's code is not well tested and these functional changes are risking a breakage. Let's stick to only documenting types for these PRs.

Reply:
Using an empty dictionary as default value is an obvious error here. See https://stackoverflow.com/a/26320938

I don't see how this change "risks a breakage". `add_rebuild_migration_yaml` is used exactly once, that is in `migration_factory` below. There, `config` and `migration_name` is set correctly.

I can create a separate PR for this little fix if you want but I usually had good experiences with a "you can always include little fixes to save time" policy.

Reply:
Moved to #2273